packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -201,6 +201,8 @@ async function runAxeTestsForBlockEditor() {
 			'link-name',
 			'listitem',
 			'region',
+			'aria-required-children',
+			'aria-required-parent',
```

Comment:
I see that `frame-title` is another new rule added that causes one of the tests to fail. So:
```suggestion
			'aria-required-parent',
			'frame-title',
```
should do the trick.